scripts/check-ts.js

```diff
@@ -61,6 +62,13 @@ for (let index = 0; index < filePaths.length; index++) {
         linesWithArrayType.push(`${filePath}#${index}: ${line}`);
       } else if (line.includes("!.")) {
         linesWithPotentialNullOrUndefined.push(`${filePath}#${index}: ${line}`);
+      } else if (
+        line.includes("@eclipse-sirius/sirius-components") &&
+        line.includes("/dist")
+      ) {
+        lineWithSiriusComponentsImportWithDist.push(
+          `${filePath}#${index}: ${line}`
+        );
```

Comment:
You need to add a check below to make the build fail in case of error

Reply:
Done